src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
```

Comment:
- [ ] Please follow the ducks pattern in naming your action types. You should have something like 👇 
```
const ADD_BOOK = 'bookstore/books/ADD_BOOK
```